assets/rollup.config.js

```diff
@@ -20,7 +20,9 @@ module.exports = {
     input: 'src/scripts/index.js',
     plugins: [
         alias({
-            leaflet: path.resolve(__dirname, 'node_modules/leaflet/dist/leaflet-src.esm.js')
+            entries: [
+                {find: 'leaflet', replacement: path.resolve(__dirname, 'node_modules/leaflet/dist/leaflet-src.esm.js')}
```

Comment:
I recall there being errors regarding leaflet when I tried to update rollup plugins a couple weeks ago. Let me scratching my head... wouldn't have guessed that's what needed to change.

Reply:
I had to read the release notes twice and the PR which introduce the changes ... it's not obvious.